src/osd/scheduler/mClockScheduler.cc

```diff
@@ -387,6 +398,29 @@ void mClockScheduler::update_configuration()
 
 void mClockScheduler::dump(ceph::Formatter &f) const
 {
+  std::ostringstream out;
+
+  // Display queue sizes
```

Comment:
nit: we should modify https://github.com/ceph/ceph/blob/08194cb2a5c80675c7ac8c0ce0dfe0cc0ce1a84f/src/osd/OSD.cc#L3906 to say "dump op queue state"

Reply:
Changed according to your suggestion.